server/routes/maintainingReports/coordinator.ts

```diff
@@ -901,4 +995,28 @@ export default class CoordinatorRoutes {
     const location = removalRequest ? paths.viewStatements(reportId) : paths.viewReport(reportId)
     return res.redirect(location)
   }
+
```

Comment:
3 functions to manage data in session, specifically in req.session.incidentReport. 
The data will an array of objects with each object representing a specific report and will contain the reportId. Setting, getting and deleting from session will be done by comparing the reportId in the url to that in session object. Targeting by reportId will enable user to have multiple reports open in different browser tabs without the session data for one report affecting the session data of another.